src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-card flat id="AR-step-4-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="Person's legal name"
+            hint="Legal name of authorized person"
```

Comment:
This label/hint/rule was taken from business-filings-ui but they are different in bcrs-business-create-ui...

What should these be in business-edit-ui? Let's change these to the requirements. (We can worry about passing more props to set this per project in a future version of this component.)